mathics/builtin/makeboxes.py

```diff
@@ -369,10 +369,6 @@ class MakeBoxes(Builtin):
     #> (a <> b)[x]
      : String expected.
      = (a <> b)[x]
-    """
-
-    # TODO: Convert operators to appropriate representations e.g. 'Plus' to '+'
-    """
     >> \\(a + b\\)
```

Comment:
I have been looking at this and notice a couple of small formatting changes. If this docstring were marked as a raw string, e.g. `r""" ... """` we would need all of the ugly and confusing double backslashes `\\` everywhere..

On line 333: 

```
      <dd>is a low-level formatting primitive that converts $expr$
        to box form, without evaluating it.
```

has a hard line break which the current homegrown formatter doesn't ignore. And it would be nice to add an extra space between and end `</dt>` and the next starting `<dd>`. 

These aren't strictly about this PR, but I happen to notice since you happen to be in the area.

Reply:
Indeed. Also, I was thinking about to patching the homegrown formatter to handle these line breaks, and then avoiding that flake8 complains about long lines. But one step at the time...

Reply:
Fixed in [f6f3d1f](https://github.com/Mathics3/mathics-core/pull/545/commits/f6f3d1fc29ed338d92829377f77d6a521e099129)